src/staverepetition.ts

```diff
@@ -13,13 +13,6 @@ export class Repetition extends StaveModifier {
     return Category.Repetition;
   }
 
-  static TEXT_FONT: Required<FontInfo> = {
-    family: Font.SERIF,
-    size: Tables.NOTATION_FONT_SCALE / 3,
```

Comment:
It used to be 1/3 of the NOTATION_FONT_SCALE. So I assume you just hardcode a number now in the CommonMetrics?

Reply:
yes